src/archive.cpp

```diff
@@ -178,6 +178,23 @@ namespace zim
     }
   }
 
+  Entry Archive::getRandomEntry() const {
+    auto frontEntryCount = m_impl->getFrontEntryCount().v;
+    int watchdog = 42;
+    while(--watchdog) {
+      auto idx = ((double)rand() / ((double)RAND_MAX+1)) * frontEntryCount;
+      auto entry = getEntryByTitle(idx);
+      auto item = entry.getItem(true);
+      std::cerr << item.getPath() << "=>" << item.getMimetype() << std::endl;
```

Comment:
Do we need this bare `std::cerr` here? Maybe wrap it in a logging statement?

Reply:
I've removed the line.